### ISSUE TITLE
I have some questions and would like to seek your clarification

### ISSUE THREAD

**Issue by 1DJ1127** [Dylan-H-Wang/skin-sm3#1]
After reading your article, I have some questions and would like to seek your clarification. My questions are as follows.：
1. Were the SM3 pre-training weights used in your paper obtained through pre-training on the Derm7pt dataset or the ImageNet dataset?
2. From reading the article, I understand that the SM3 method does not utilize the dataset labels during the entire process but instead employs a self-supervised approach to learn features based solely on the images themselves. Then, during the "fine-tuning" stage, the specific label data is used for the classification task. Is this understanding correct? How is the fine-tuning conducted?
3. In your article, you mentioned using "Label Projection Heads" to project features extracted from images into a label-specific embedding space to generate pseudo-label features. I don't quite understand how this projection method is implemented, given that specific label data information is not used. How are the pseudo-labels generated through this projection? What is the form of these pseudo-label features, and what information do they contain?

**Comment by Dylan-H-Wang** [comment]
Hi @1DJ1127 ,

Thank you for your interests!

1. SM3 pre-training weights were obtained by pre-training the model using SM3 on the Derm7pt  dataset.
2. Yes, your understanding is correct. The linear probe protocol is a standard approach to evaluate SSL algorithms by freezing all layers except the final fully connected (FC) layer. For more details, please refer to Section 4.1 in [MoCo](https://arxiv.org/pdf/1911.05722) and Supplementary A in [MAE](https://arxiv.org/pdf/2111.06377).
3. The pseudo-labeling approach was inspired by the [DeepCluster](https://arxiv.org/pdf/1807.05520) paper, which demonstrates that unsupervised clustering can generate pseudo-labels to train neural networks in a self-reinforcing manner. In our work, we extend this idea to a multi-label, multi-class setting. Features before the "Label Projection Heads" capture general image information, while the "Label Projection Heads" distill label-specific features.

Kind regards

**Comment by 1DJ1127** [comment]
Thank you very much!!! Hope you have a nice day!!!

**Comment by 1DJ1127** [comment]
Hi @Dylan-H-Wang ,
I still have a question: can the SM3 pre-training weights you provided only be used in the ResNet network, or can they also be used in other feature extraction networks?

Kind regards,

**Comment by Dylan-H-Wang** [comment]
> Hi @Dylan-H-Wang , I still have a question: can the SM3 pre-training weights you provided only be used in the ResNet network, or can they also be used in other feature extraction networks?
> 
> Kind regards,

Yes, it can only be used for ResNet, specifically ResNet-50.

You can pre-train using other models when the code is released

Kind regards
